### DISCUSSION TITLE
Interrupt/Deactivate a Form

### DISCUSSION THREAD

**Discussion by WissamAntoun** [botfront/botfront#947] [Q&A]
I have been following the guide written here https://botfront.io/blog/a-no-code-approach-to-rasa-forms-form-action-and-form-policy. I know it's old (and has a wrong video in the last resume branch). I was trying to deactivate a form that I created but couldn't find that in the "Loop". I search in the Docs but couldn't find any mention of it.

So how can I deactivate a form?

**Comment by castigliana** [comment]
I have the same issue related to resuming a form if user digressed. In the "Adjusting our stories" section of the [Documentation ](https://botfront.io/blog/a-no-code-approach-to-rasa-forms-form-action-and-form-policy) video is incorrect related to resume branch.
